pkg/util/shoot_clients.go

```diff
@@ -102,3 +107,27 @@ func NewClientsForShoot(ctx context.Context, c client.Client, namespace string,
 		version:           shootVersion,
 	}, nil
 }
+
+// NewChartRendererForShoot creates a new chartrenderer.Interface for the shoot cluster.
+func NewChartRendererForShoot(version string) (chartrenderer.Interface, error) {
+	v, err := VersionInfo(version)
+	if err != nil {
+		return nil, err
+	}
+	return chartrenderer.New(engine.New(), &chartutil.Capabilities{KubeVersion: v}), nil
+}
+
+var versionRegex = regexp.MustCompile(`^(\d+\.)?(\d+\.)?(\d+)$`)
```

Comment:
Should we rather use [Masterminds/semver](https://github.com/gardener/gardener/blob/master/Gopkg.toml#L119-L121) like in gardener?

Reply:
Good idea. In fact, `Mastermind/semver` was already used by `VersionMajorMinor`, so I refactored `VersionInfo` accordingly, moved it to `shoot.go`, and added a test.